ios/FluentUI.xcodeproj/project.pbxproj

```diff
@@ -1625,6 +1625,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = 53FC90F825673626008A06FD /* FluentUI_framework.xcconfig */;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
Can we add to FluentUI_framework.xcconfig instead so we don't have to duplicate for each config in the very difficult-to-read .pbxproj file?

Reply:
Great suggestion. I'll do it.